internal/database/sqlcommon/identity_sql.go

```diff
@@ -0,0 +1,250 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlcommon
+
+import (
+	"context"
+	"database/sql"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/hyperledger/firefly/internal/i18n"
+	"github.com/hyperledger/firefly/internal/log"
+	"github.com/hyperledger/firefly/pkg/database"
+	"github.com/hyperledger/firefly/pkg/fftypes"
+)
+
+var (
+	identityColumns = []string{
+		"id",
+		"did",
+		"parent",
+		"itype",
+		"namespace",
+		"name",
+		"description",
+		"profile",
+		"messages_claim",
+		"messages_verification",
+		"messages_update",
+		"created",
+		"updated",
```

Comment:
Note the updated is due to the fact that there's a `PATCH` on these, with the new `IdentityUpdate` flow.